libs/community/tests/unit_tests/vectorstores/test_sqlserver.py

```diff
@@ -307,23 +305,38 @@ def test_sqlserver_delete_text_by_id_invalid_ids_provided(
 
     result = store.delete(["100000"])
     # Should return False since given id is not in DB
-    if not result:
-        pass
+    assert not result
 
 
 def test_sqlserver_delete_text_by_id_no_ids_provided(
     store: SQLServer_VectorStore,
     texts: List[str],
     metadatas: List[dict],
 ) -> None:
-    """Test that delete API deletes texts by id."""
+    """Test that delete API deletes data in vectorstore
+    when `None` is provided as the parameter."""
 
     store.add_texts(texts, metadatas)
-
     result = store.delete(None)
+
+    # Should return True, since None is provided,
+    # all data in vectorstore is deleted.
+    assert result
```

Comment:
is it possible to test the vector store is empty to validate ALL data is deleted?

Reply:
Updated.